### PR TITLE
Add documentation on authorizated requests to the API

### PR DESCRIPTION
**Pull request by bnd5k**
Relates to this PR for the API: https://github.com/Ragtagteam/cta-aggregator/pull/69

### CODE REVIEW

File: source/includes/_authorization.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+If you have API credentials, then you can create resources or update the resources you've created.
+
+When updating a resource, you cannot make an update to a resource that was created by another contributor.
+The API will reject your request.
+
+HTTP Method   | Authentication Required? | Notes
+--------- |  ----------- |  -----------
+Get | No | No authentication required to read any resource(s).
+POST | Yes | Authentication required but anyone with API credentials can create a resource.
```
Comment (AaronSofaer) [inline source/includes/_authorization.md]: Nit: comma before the 'but'.

File: source/includes/_authorization.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PUT | Yes | Request will be accepted if you created the resource you are updating. 
+DELETE | N/A | Deleting not currently permitted.  To get a resource removed, conctact the Ragtag team at <a href='mailto:ctaaggregator@ragtag.org'>ctaaggregator@ragtag.org</a>).
```
Comment (AaronSofaer) [inline source/includes/_authorization.md]: Nit: "Deleting resources is not".

**Review by AaronSofaer — APPROVED** [review]
Two grammar nits, looks good otherwise.

**Comment by bnd5k** [comment]
Cool.  Typos fixed.

I'll merge this once the PR for the API passes review.

**Pull request merged**